acceptance-tests/src/main/kotlin/org/ostelco/at/common/StripePayment.kt

```diff
@@ -43,6 +43,25 @@ object StripePayment {
         return token.card.id
     }
 
+    fun getDefaultSourceForCustomer(customerId: String) : String {
```

Comment:
Why did we reimplement the functions that we have created APIs for?

Reply:
It is used to check whether the "setDefaultSource" API actually works as intended (by obtaining the information directly from Stripe). But bad naming perhaps?